src/hooks/useTxDetails/useTxDetails.ts

```diff
@@ -80,13 +80,13 @@ export const useTxDetails = (txId: string, activeAsset?: Asset): TxDetails => {
   )
 
   // stables need precision of eth (18) rather than 10
-  const feeAsset = useSelector((state: ReduxState) =>
+  const feeAsset: Asset | undefined = useSelector((state: ReduxState) =>
```

Comment:
change these to `useAppSelector` and they're already typed, also update the type def of the selector and the return type will be correct